client/src/components/StepWizard/WizardNav.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import styled from "styled-components";
 import { Link } from "react-router-dom";
+import { mq } from "../../constants/theme";
```

Comment:
Please use absolute imports. Should be `constants/theme`

Reply:
Ok I will adjust it.